platform/jvm/capture/build.gradle.kts

```diff
@@ -97,26 +83,14 @@ android {
 }
 
 // Rust cargo build toolchain
-cargo {
-    libname = "capture"
-    extraCargoBuildArguments = listOf("--package", "capture")
+cargoNdk {
+    librariesNames = arrayListOf("libcapture.so")
+    extraCargoBuildArguments = arrayListOf("--package", "capture")
     module = "../.."
-    targetDirectory = "../../../target"
-    targets = listOf("arm64", "x86_64")
-    pythonCommand = "python3"
-    exec = { spec, _ ->
-        // enable 16 KB ELF alignment on Android to support API 35+
-        spec.environment("RUST_ANDROID_GRADLE_CC_LINK_ARG", "-Wl,-z,max-page-size=16384")
-    }
-}
-
-// workaround bug in rust-android-gradle plugin that causes .so to not be available on app launch
-// see: https://github.com/mozilla/rust-android-gradle/issues/118#issuecomment-1569407058
-tasks.whenTaskAdded {
-    if ((this.name == "mergeDebugJniLibFolders" || this.name == "mergeReleaseJniLibFolders")) {
-        this.dependsOn("cargoBuild")
-        this.inputs.dir(layout.buildDirectory.dir("rustJniLibs/android"))
-    }
+    targetDirectory = "./target"
```

Comment:
how is it able to find the target here? doesn't this run inside capture-sdk/platform/jvm ?

Reply:
`module` specifies ../.., so this becomes ../../target